x-pack/plugins/cases/common/constants/application.ts

```diff
@@ -12,9 +12,11 @@ import { CASE_VIEW_PAGE_TABS } from '../types';
  */
 
 export const APP_ID = 'cases' as const;
-/** @deprecated Please use FEATURE_ID_V2 instead */
+/** @deprecated Please use FEATURE_ID_V3 instead */
 export const FEATURE_ID = 'generalCases' as const;
+/** @deprecated Please use FEATURE_ID_V3 instead */
 export const FEATURE_ID_V2 = 'generalCasesV2' as const;
+export const FEATURE_ID_V3 = 'generalCasesV3' as const;
```

Comment:
@azasypkin - I don't remember if I had asked this, but do you know if we default collect telemetry of what role versions users are on?